internal/resources/grafana/resource_alerting_notification_policy_test.go

```diff
@@ -139,6 +139,57 @@ func TestAccNotificationPolicy_disableProvenance(t *testing.T) {
 	})
 }
 
+func TestAccNotificationPolicy_inOrg(t *testing.T) {
+	testutils.CheckOSSTestsEnabled(t, ">=9.1.0")
+
+	var policy models.Route
+	var org models.OrgDetailsDTO
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: testutils.ProviderFactories,
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			orgCheckExists.destroyed(&org, nil),
+			alertingNotificationPolicyCheckExists.destroyed(&policy, &org),
+		),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNotificationPolicyInOrg(),
+				Check: resource.ComposeTestCheckFunc(
+					orgCheckExists.exists("grafana_organization.test", &org),
+					alertingNotificationPolicyCheckExists.exists("grafana_notification_policy.test", &policy),
+					checkResourceIsInOrg("grafana_notification_policy.test", "grafana_organization.test"),
+				),
+			},
```

Comment:
Add another step to check that destroying within the org works. See other resources for example